graphql-api/src/main/resources/stream-registry.graphql

```diff
@@ -246,6 +247,16 @@ type Consumer {
     binding: ConsumerBinding
 }
 
+############ Session ############
+
+type Session {
+    id: ID
```

Comment:
I think a more natural ordering would be id, secret, expiresAt, producerBindings, consumerBindings

Reply:
👍